### DISCUSSION TITLE
CHM Documentation

### DISCUSSION THREAD

**Discussion by kdub-nz** [wtertinek/Linq2Acad#28] [Ideas]
Could the  /master/docs/api/  contents be converted to a CHM help file.

Having a CHM that is current with API details and sample code would be more convenient than the current folder based system.

![image](https://user-images.githubusercontent.com/2830167/208541296-615aef9f-6c61-4f65-ad2d-11559174d957.png)

**Comment by wtertinek** [comment]
Hi kdub-nz,

it is intended that the documentation is viewed in the browser. [This](https://github.com/wtertinek/Linq2Acad/blob/master/docs/api/Index.md#Linq2Acad-Namespace) is the main page. It is not necessary to navigate through the files. Or is it necessary for you to have the files offline?

Br,
Wolfgang

**Reply by benkoshy** [reply]
In my experience, when using the AutoCAD .net documentation - i use the offline version. They have some handy search functionalities which is cumbersome online etc. and I suspect .chm request is due to an existing way of operating. 

I personally would not need it, but if it was there, I would use it.

I searched for a github action which would build the chm file whenever there's a commit to the master branch etc. but could not find one - granted, I did not look too hard either.

my two cents.

**Comment by wtertinek** [comment]
Currently, the Markdown documentation is created from the XML comments using [Sandcastle Help File Builder](https://ewsoftware.github.io/SHFB/html/bd1ddb51-1c4f-434f-bb1a-ce2135d3a909.htm). Sandcastle can create CHM files, but the Markdown output needs significant post-processing and this post-processing would be missing in the CHM file. I have not yet found a free converter that can convert Marktown to CHM. Therefore I don't see any possibility to create a CHM file at the moment.

**Reply by kdub-nz** [reply]
I'm pretty sure that Doxygen can input Markdown .md files.
Not sure if the output would be suitable though.

Don't go out of your way to satisfy me , If I decide to use the library I'll probably build my own CHM.
